index.html 

```diff
@@ -3,25 +3,33 @@
 	<title>Bubble Tea site</title>
 </head>
 <body>
-	<div class="topnav">
-		<img src="" alt="logo" style="width:600px; height:400px"/>
-		<a class="active" href=#home">Home</a>
-		<a href=#news">News</a>
-		<a href=#about">About</a>
-		<a href=#contact”>Contact</a>
-	</div
-	<div>
-		<img src="" alt="description of image">
-	</div>
+	<nav>
```

Comment:
Can you add a `<header>` above this `<nav>`, and within this new tag add a `h1` with "Hello Word" as the title?